CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,65 @@
+# Code of Conduct
+
+Like the technical community as a whole, the various ESLint teams and communities are made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+
+This isn't an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it's intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+* **Be friendly and patient.** We understand that we all have different levels of experience or knowledge in many diverse fields, be it technical or non technical in nature. We also have areas of knowledge we are eager to expand, and we want to be a community where people can not only contribute, but feel comfortable to ask questions as well and learn along the way. If someone is wrong, or says something accidentally offensive, respond with patience and try to keep it polite and civil. Remember that we all were newbies at one point.
+
+* **Be welcoming.** We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+
+* **Be considerate.** Your work will be used by other people, and you in turn will depend on the work of others. Any decision you make will affect users and colleagues, and you should take those consequences into account when making decisions. Remember that we're a world-wide community, so you might not be communicating in someone else's primary language.
```

Comment:
Not sure if "world-wide" or "worldwide" is better here. Prescriptively I would go with "worldwide" but "world-wide" might be seeing more usage in today's lexicon...